app/assets/javascripts/controllers/security_group/security_group_form_controller.js

```diff
@@ -12,11 +12,11 @@ ManageIQ.angular.app.controller('securityGroupFormController', ['$http', '$scope
     $scope.securityGroupModel.description = "";
     $scope.newRecord = true;
   } else {
-    miqService.sparkleOn();
-
-    $http.get('/security_group/security_group_form_fields/' + securityGroupFormId)
-      .then(getSecurityGroupFormData)
-      .catch(miqService.handleFailure);
+    API.get("/api/security_groups/" + securityGroupFormId + "?attributes=cloud_tenant").then(function(data) {
+      $scope.securityGroupModel = data;
```

Comment:
What's the reason for assigning the entire response `data` to the model? I don't think this is right.

This change should be almost identical to what you had below in  `getSecurityGroupFormData` that you deleted.
```diff
-  function getSecurityGroupFormData(response) {
-    var data = response.data;
-
-    $scope.afterGet = true;
-    $scope.securityGroupModel.name = data.name;
-    $scope.securityGroupModel.description = data.description;
-    $scope.securityGroupModel.cloud_tenant_name = data.cloud_tenant_name;
-    $scope.modelCopy = angular.copy( $scope.securityGroupModel );
-    miqService.sparkleOff();
-  }
```

So it should be something like this:
```diff
+    $scope.afterGet = true;
+    $scope.securityGroupModel.name = data.name;
+    $scope.securityGroupModel.description = data.description;
+    $scope.securityGroupModel.cloud_tenant_name = data.cloud_tenant.name;
+    $scope.modelCopy = angular.copy( $scope.securityGroupModel );
+    miqService.sparkleOff();
```

This is assuming that `data` contains `name` and `description` values